Cargo.toml

```diff
@@ -3,26 +3,26 @@ name = "yew_webview_bridge"
 description = "2-way communcation bridge between web-view and yew"
 keywords = ["yew", "web-view"]
 license = "MIT OR Apache-2.0"
-version = "0.1.0"
+version = "0.2.0"
 authors = ["Maximilian Goisser <goisser94@gmail.com>"]
 edition = "2018"
 repository = "https://github.com/hobofan/yew_webview_bridge"
 readme = "README.md"
 
 [dependencies]
 serde = { version = "1", features = ["derive"] }
-serde_json = { version = "1.0.52" }
-rand = { version = "=0.6.5", features = ["wasm-bindgen"] }
+serde_json = { version = "1.0" }
+rand = { version = "0.7", features = ["wasm-bindgen"] }
```

Comment:
If there was an exact version specified, I'm sure there was a good reason for it.

IIRC, newer versions of rand caused an error (visible on the console) when used in the browser (at least in the environment provided by webview on OSX).

Would be good if you can reverse that change, and I'll add a tracking issue for it.

Reply:
Okay, no worries, I'll put back the exact version for `rand`.

Reply:
hey @hobofan I'm getting the following stacktrace while experimenting on using the new `rand`:

```
[Error] Unhandled Promise Rejection: Error: Out of bounds memory access (near '...e__h0d03c6547e5e0185(arg0, arg1, addHeap...')
	<?>.wasm-function[<T as core::convert::Into<U>>::into::h8e2dc1dc8cfddf13] (data:text/html,%3C…%3E:271:135)
	<?>.wasm-function[ppv_lite86::generic::dmap2::h9a1309b08a243710]
	<?>.wasm-function[rand_chacha::guts::refill_wide::ha379d3985d906d93]
	<?>.wasm-function[<rand_chacha::chacha::ChaCha20Core as rand_core::block::BlockRngCore>::generate::h228564273ae9a3c9]
	<?>.wasm-function[<rand::rngs::adapter::reseeding::ReseedingCore<R,Rsdr> as rand_core::block::BlockRngCore>::generate::h6885e5a5fa188921]
	<?>.wasm-function[rand_core::block::BlockRng<R>::generate_and_set::h6505ddccc6a5f801]
	<?>.wasm-function[<rand_core::block::BlockRng<R> as rand_core::RngCore>::next_u32::h8b977afb2b63748d]
	<?>.wasm-function[rand::distributions::integer::<impl rand::distributions::Distribution<u32> for rand::distributions::Standard>::sample::hc3ae3caece8847d4]
	<?>.wasm-function[rand::Rng::gen::hb1a7fdc6a5707799]
	<?>.wasm-function[yew_webview_bridge::Message<T>::generate_subscription_id::h956e2c2444c72c04]
```

Was this the same issue you were encountering on OSX? Was it reported?

Reply:
It's been some time, but I don't think it was that error, but a more proper one, like the one when you run into a `unimplemented!()`/`unreachable!()`.